Tools/ApplicationDebugger/array-transform/README.md

```diff
@@ -143,18 +143,18 @@ Perform the following steps:
 
 2.  Run the program:
     ```
-    $ ./array-transform <device>
+    $ ./array-transform
     ```
-    > Note: `<device>` is the type of the device type to offload the kernel.
-    > Use `cpu`, `gpu`, or `accelerator` to select the CPU, GPU, or the
-    > FPGA emulator device, respectively.  E.g.:
+    > Note: to specify a device type to offload the kernel, use
```

Comment:
Nit: use *the* ONEAPI... environment variable

Reply:
fixed.